src/txn/txn.c

```diff
@@ -1202,6 +1350,15 @@ __wt_txn_prepare(WT_SESSION_IMPL *session, const char *cfg[])
         if (WT_IS_METADATA(op->btree->dhandle))
             continue;
 
+        /*
+         * Logged table updates should never be prepared. As these updates are immediate durable, it
+         * is not possible to rollback them in case if the prepared transaction is rollback-ed.
```

Comment:
Should read:

`Logged table updates should never be prepared. As these updates are immediately durable, it is not possible to roll them back if the prepared transaction is rolled back.`

Reply:
Fixed.